### PR TITLE
Fix autosize text and extract number panel

### PR DESCRIPTION
**Pull request by ademar111190**
How it looks like

|Before|After|
|---|---|
|error|fixed|

### CODE REVIEW

**Review by ubbabeck — APPROVED** [review]
Looks good.  Minor remark I think you have a typo in the filename shouldn't it be `number_panel.dart` ?

Wait for roei or erdems approval before mergning 😄

**Review by erdemyerebasmaz — APPROVED** [review]

File: lib/routes/podcast/boost.dart
Comment (erdemyerebasmaz) [inline lib/routes/podcast/boost.dart:209]: maxLines was missing here, was that the issue?

**Pull request merged**
